packages/orchestration/src/facade.js

```diff
@@ -37,17 +29,30 @@ const mockLocalChainInfo = {
 
 /**
  * @param {Remote<LocalChain>} localchain
+ * @param {ReturnType<
+ *   typeof import('./exos/local-chain-account-kit.js').prepareLocalChainAccountKit
+ * >} makeLocalChainAccountKit
  * @returns {Chain}
  */
-const makeLocalChainFacade = localchain => {
+const makeLocalChainFacade = (localchain, makeLocalChainAccountKit) => {
   return {
     /** @returns {Promise<ChainInfo>} */
     async getChainInfo() {
       return mockLocalChainInfo;
     },
 
     async makeAccount() {
-      const account = await E(localchain).makeAccount();
+      const lowLevelAccountP = E(localchain).makeAccount();
```

Comment:
naming by "level" is a smell.

`makeLocalChainAccountKit` is making something that does OrchestrationAccount and more. Let's rename it to `LocalOrchestrationAccount` like the `RemoteOrchestrationAccount`.

Up to you whether this PR. I'm happy to do it. If you want to land this as-is please at least leave a TODO comment.